src/platform/site-wide/sass/minimal.scss

```diff
@@ -2,13 +2,20 @@
 @import "modules/m-overrides";
 
 @import "~uswds/src/stylesheets/core/variables";
+// *** Formation override ***
+@import "formation-overrides/variables";
+
 @import "~uswds/src/stylesheets/core/base";
 @import "~uswds/src/stylesheets/core/fonts";
+
 @import "~uswds/src/stylesheets/core/utilities";
+// *** Formation override ***
+@import "formation-overrides/utilities";
```

Comment:
This is just an example of how we'd convert the rem to the correct USWDS value.

I created a new folder in `sass/formation-overrides/`. The minimal stylesheet imports that has a `rem` value somewhere is duplicated and the `uswds-override` function applied to it. That override stylesheet is loaded directly after the one it's replicating.